components/webgpu/ipc_messages/recv.rs

```diff
@@ -2,7 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
```

Comment:
How about "from_script.rs" for this file name. I think it's useful to think about messages as either going to some place or coming from it.

Reply:
The problem is that wgpu thread receives messages from multiple threads (constellation, script_thread, dom?), so the most appropriate description is receiving (in wgpu thread).